src/autodiff.jl

```diff
@@ -115,32 +119,21 @@ struct Jacobian{VI, VT, MT, SMT, VP, VD, SubT, SubD}
             J = CUSPARSE.CuSparseMatrixCSR(J)
         end
         t1s{N} = ForwardDiff.Dual{Nothing,Float64, N} where N
-        if isa(type, StateJacobian)
-            x = VT(zeros(Float64, 2*nvbus))
-            t1sx = A{t1s{ncolor}}(x)
-            t1sF = A{t1s{ncolor}}(zeros(Float64, nmap))
-            t1sseeds = A{ForwardDiff.Partials{ncolor,Float64}}(undef, nmap)
-            _init_seed!(t1sseeds, coloring, ncolor, nmap)
-            compressedJ = MT(zeros(Float64, ncolor, nmap))
-            varx = view(x, map)
-            t1svarx = view(t1sx, map)
-        elseif isa(type, ControlJacobian)
-            x = VT(zeros(Float64, npbus + nvbus))
-            t1sx = A{t1s{ncolor}}(x)
-            t1sF = A{t1s{ncolor}}(zeros(Float64, length(F)))
-            t1sseeds = A{ForwardDiff.Partials{ncolor,Float64}}(undef, nmap)
-            _init_seed!(t1sseeds, coloring, ncolor, nmap)
-            compressedJ = MT(zeros(Float64, ncolor, length(F)))
-            varx = view(x, map)
-            t1svarx = view(t1sx, map)
-        else
-            error("Unsupported Jacobian type. Must be either ControlJacobian or StateJacobian.")
-        end
-
-        VP = typeof(t1sseeds)
-        VD = typeof(t1sx)
-        return new{VI, VT, MT, SMT, VP, VD, typeof(varx), typeof(t1svarx)}(
-            J, compressedJ, coloring, t1sseeds, t1sF, x, t1sx, map, varx, t1svarx
+        init_seed_kernel! = _init_seed!(KA.CPU(), 4)
```

Comment:
I would maybe add a comment that we always do the seeding on the CPU

Reply:
Done.